README.md

```diff
@@ -1,10 +1,12 @@
 # Open WebRTC Toolkit Native SDK
 
 ## Introduction
-The Open WebRTC Toolkit client SDK for native Windows/Linux/Android/iOS applications is built upon the W3C WebRTC standard to accelerate the development of real time communication applications on these platforms. It supports peer to peer and conference mode communication working with Open Media Stream MCU server.
+The Open WebRTC Toolkit(OWT) client SDK for native Windows/Linux/Android/iOS applications is built upon the W3C WebRTC standard to accelerate the development
+of real time communication applications on these platforms. It supports peer to peer communication, and conference mode communication working with
+Open WebRTC Toolkit MCU server.
 
-- Supported Windows platform: Windows 7 and above.
-- Supported Linux platform: Ubuntu 16.04.
+- Supported Windows platform: Windows 8 and above.
+- Supported Linux platform: Ubuntu 16.04 & 18.04.
```

Comment:
Not sure if 16.04 is supported, but 20.04 should be supported as well.

Reply:
Will add this when docker file is created with Ubuntu 20.04 base image.